docs/symphony-book/agent/flatcar.md

```diff
@@ -27,7 +27,7 @@
 
 1. Upload the `.raw` file to a GitHub release folder.
 
-   The ignition file under the repo uses a temporary GitHub release at `https://github.com/Haishi2016/Vault818/releases/download/vtest/piccolo.raw`. To use a different GitHub repo, you'll need to update the ignite.ign file and update the source folder to the repo you want to use.
+   The ignition file under the repo uses a temporary GitHub release at `https://github.com/eclipse-symphony/symphony/releases/download/vtest/piccolo.raw`. To use a different GitHub repo, you'll need to update the ignite.ign file and update the source folder to the repo you want to use.
```

Comment:
@Haishi2016 Caught this unchanged reference in my global search+replace. Do you want it to continue pointing to Haishi2016 or updated?

Reply:
This should be pointing to Haishi2016 for now, we haven't figured out the best way to release this. Thanks!

Reply:
Can we update this? I want to close this PR ASAP.

Reply:
Sorry, I thought I had. I'll make this change now.

Reply:
Edit -- this was fixed, and the comment thread is on an outdated version of the file. Now the flatcar.md file is unchanged in the PR, and I double checked that the branch has the right github url:

<img width="1109" alt="image" src="https://github.com/eclipse-symphony/symphony/assets/13578436/95fa7539-f65b-4b08-8615-b4dfb5c941b5">

Now I'm just resolving the merge conflict and we should be good to go.